testdata/std.makeArray_noninteger_big.jsonnet

```diff
@@ -1 +1 @@
-std.makeArray(1e100, error "shouldn't happen")
+std.makeArray(1e100, function(i) i)
```

Comment:
The args are forced now, so the fact that 1e100 is too large is not discovered before the error is evaluated. The cpp version always had that behavior.  The semantics are ambiguous (because it's an error either way and the semantics does not distinguish between different kinds of errors).